core/src/main/java/com/netease/arctic/io/reader/BaseIcebergDataReader.java

```diff
@@ -107,6 +107,19 @@ public CloseableIterable<T> readData(ArcticFileScanTask task) {
     return iterable;
   }
 
+  public CloseableIterable<T> readDataWithoutApplyPosDelete(ArcticFileScanTask task) {
```

Comment:
the new method makes this class very strange. functions use the old `readData` method still have bad performace like flink reader?

Reply:
The readData method is used by original iceberg read